openshift-kube-apiserver/admission/customresourcevalidation/route/validate_route.go

```diff
@@ -45,18 +41,21 @@ func toRoute(uncastObj runtime.Object) (*routev1.Route, field.ErrorList) {
 }
 
 type routeV1 struct {
+	sarc           authorizationv1client.SubjectAccessReviewInterface
```

Comment:
I think that you will need a function here as well.

Reply:
same for `secretsGetter`

Reply:
Addressed